src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work.");

```

Comment:
The message in the `doWork` method should indicate the specific type of Machine that is working. Consider making the message more informative, for example: 'Truck is doing its work.'